agenda/presentation/src/main/java/com/nibalk/tasky/agenda/presentation/utils/LocalDateExt.kt

```diff
@@ -0,0 +1,21 @@
+package com.nibalk.tasky.agenda.presentation.utils
+
+import java.time.LocalDate
+
+fun LocalDate.getSurroundingDays(
+    before: Int = 12,
+    after: Int = 17
+): List<LocalDate> {
+    val days = mutableListOf<LocalDate>()
+    // Add days before
+    for (i in before downTo 1) {
+        days.add(this.minusDays(i.toLong()))
+    }
+    // Add current day
+    days.add(this)
+    // Add days after
+    for (i in 1..after) {
+        days.add(this.plusDays(i.toLong()))
+    }
+    return days
+}
```

Comment:
A quicker way would be this (that doesn't mean that yours is worse 😄 ):
```
val startDate = this.minusDays(before)
return (0 until (before + after)).map { startDate.plusDays(it) }
```
I would also make sure that you don't return mutable lists in such functions, since even though you hide it behind a `List<LocalDate>`, it's still technically mutable under the hood. You can simply convert it with `toList()`